engine/org.eclipse.birt.report.engine/src/org/eclipse/birt/report/engine/nLayout/area/style/TextStyle.java

```diff
@@ -62,98 +87,218 @@ public TextStyle(TextStyle style) {
+	 *
+	 * @param align text alignment
+	 */
 	public void setAlign(CSSValue align) {
 		this.align = align;
 	}
 
+	/**
+	 * Set the text color
+	 *
+	 * @param color color of the text
+	 */
 	public void setColor(Color color) {
 		this.color = color;
 	}
 
+	/**
+	 * Get the text color
+	 *
+	 * @return Return the text color
+	 */
 	public Color getColor() {
 		return color;
 	}
 
+	/**
+	 * Set the font size
+	 *
+	 * @param fontSize font size
+	 */
 	public void setFontSize(int fontSize) {
 		this.fontSize = fontSize;
 	}
 
+	/**
+	 * Set the font info
+	 *
+	 * @param fontInfo font info
+	 */
 	public void setFontInfo(FontInfo fontInfo) {
 		this.fontInfo = fontInfo;
 	}
 
+	/**
+	 * Set the letter spacing
+	 *
+	 * @param letterSpacing letter spacing
+	 */
 	public void setLetterSpacing(int letterSpacing) {
 		this.letterSpacing = letterSpacing;
 	}
 
+	/**
+	 * Set the word spacing
+	 *
+	 * @param wordSpacing word spacing
+	 */
 	public void setWordSpacing(int wordSpacing) {
 		this.wordSpacing = wordSpacing;
 	}
 
+	/**
+	 * Set the text underline
+	 *
+	 * @param underLine text is underlined
+	 */
 	public void setUnderLine(boolean underLine) {
 		this.underLine = underLine;
 	}
 
+	/**
+	 * Set the line through
+	 *
+	 * @param lineThrough text is lined through
+	 */
 	public void setLineThrough(boolean lineThrough) {
 		this.lineThrough = lineThrough;
 	}
 
+	/**
+	 * Set the overline
+	 *
+	 * @param overLine the text is overlined
+	 */
 	public void setOverLine(boolean overLine) {
 		this.overLine = overLine;
 	}
 
+	/**
+	 * Set the text direction
+	 *
+	 * @param direction text direction
+	 */
 	public void setDirection(int direction) {
 		this.direction = direction;
 	}
 
+	/**
+	 * Check if the text direction is RtL
+	 *
+	 * @return Return the check result of RtL
+	 */
 	public boolean isRtl() {
 		return direction == DIRECTION_RTL;
 	}
 
+	/**
+	 * Get the font size
+	 *
+	 * @return Return the font size
+	 */
 	public int getFontSize() {
 		return fontSize;
 	}
 
+	/**
+	 * Get the font info
+	 *
+	 * @return Return the font info
+	 */
 	public FontInfo getFontInfo() {
 		return fontInfo;
 	}
 
+	/**
+	 * Get the letter spacing
+	 *
+	 * @return Return the letter spacing
+	 */
 	public int getLetterSpacing() {
 		return letterSpacing;
 	}
 
+	/**
+	 * Get the word spacing
+	 *
+	 * @return Return the word spacing
+	 */
 	public int getWordSpacing() {
 		return wordSpacing;
 	}
 
+	/**
+	 * Check if text is underlined
+	 *
+	 * @return Return the check result
+	 */
 	public boolean isUnderline() {
 		return underLine;
 	}
 
+	/**
+	 * Check if the text use line through
+	 *
+	 * @return Return the check result
+	 */
 	public boolean isLinethrough() {
 		return lineThrough;
 	}
 
+	/**
+	 * Check if the text is underlined
+	 *
+	 * @return Return the check result
+	 */
 	public boolean isOverline() {
 		return overLine;
 	}
 
+	/**
+	 * Get the text direction
+	 *
+	 * @return Return the text direction
```

Comment:
What are the valid values?

Reply:
I haven't checked all at all normaly RTL and LTR